test/integration/suite_test.go

```diff
@@ -117,29 +124,16 @@ func TestMain(m *testing.M) {
 		fmt.Println("INFO: no existing cluster found, deploying using Kubernetes In Docker (KIND)")
 
 		builder.WithAddons(metallb.New())
-		// For some reason we've ended up using kind v0.15.0 (which by default deploys k8s v1.25)
-		// even though that
-		// * our CI runners use ubuntu-latest (which at the time of writing this comment was ubuntu20.04)
-		//   which uses kind v0.14 https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md
-		// * when used with ktf (which at the time of writing this comment was set to v0.19.0) we
-		//   should use kind v0.14 since that what ktf has set as dependency
-		//
-		// With all that said, we still managed to get kind v0.15 on our CI
-		// https://github.com/Kong/kubernetes-ingress-controller/runs/8211490522?check_suite_focus=true#step:5:6
-		// which causes issues down the line (metallb manifests using PSP which is not available
-		// in k8s v1.25+).
```

Comment:
While I like this change I feel that we'll loose the context of why we need this specific version without this comment.

Having said that: as far as I remember we don't need this anymore after https://github.com/Kong/kubernetes-testing-framework/pull/371 got merged (but we can remove this default version override in a separate PR).

Reply:
So what is you suggestion here for using the default kubernetes version? just do not specify a default version, and remove this comment, or put the comment back?

Reply:
Don't change it in this PR and create a separate one where we don't specify the default (just follow whatever the underlying `kind` uses as default, as it was before the 0.15 breakage for k8s v1.25)

Reply:
updated.